api/src/com/cloud/agent/api/storage/OVFHelper.java

```diff
@@ -113,7 +113,7 @@ public static Long getDiskVirtualSize(Long capacity, String allocationUnits, Str
                 String allocationUnits = disk.getAttribute("ovf:capacityAllocationUnits");
                 od._diskId = disk.getAttribute("ovf:diskId");
                 od._fileRef = disk.getAttribute("ovf:fileRef");
-                od._populatedSize = Long.parseLong(disk.getAttribute("ovf:populatedSize") == null ? "0" : disk.getAttribute("ovf:populatedSize"));
+                od._populatedSize = Long.parseLong(StringUtils.isBlank(disk.getAttribute("ovf:populatedSize")) ? "0" : disk.getAttribute("ovf:populatedSize"));
```

Comment:
Why not use `org.apache.commons.lang3.math.NumberUtils.toLong(String)` instead?
Then, you do not even need this inline IF

Reply:
Excellent, thanks @rafaelweingartner